subclass/Arbiter; Oath of Sanctuary .json

```diff
@@ -0,0 +1,267 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "Havels Random Finds",
```

Comment:
This json identifier is already in use: https://github.com/TheGiddyLimit/homebrew/blob/fc84c6a5e77c9b96cd694db30989bd6a7636ae00/subclass/Piggby%3B%20Oath%20of%20Ashes.json#L5

Reply:
Oh I gotta change that to? Even though thats still me?

Reply:
Yes, multiple sources can't have the same `json`. You'd need to add this to the same source, and you can only do that in the same file.

Reply:
I just changed it. Turned green though so im good right?